source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -401,9 +401,9 @@ Testing CA Success
 Client connections
 ******************
 
-.. versionadded:: v2.3.4 
-
 Dovecot uses default system CAs for outgoing connections.
+The :ref:`setting-ssl_client_ca_dir` is preferred because it uses less memory.
```

Comment:
Could indicate that using System CAs is default behavior.

Reply:
I changed the text here. I'm not actually sure how exactly the openssl system CAs work. Are they built-in or read from some default directory or what..